exercises/en/test_04_12.py

```diff
@@ -0,0 +1,18 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+    assert "ikea_shoe_rack" in __solution__, "Make sure you are naming your result 'ikea_shoe_rack'."
```

Comment:
Can you say make sure you are naming your _**object**_ ......  for all of these?

Reply:
Done!